.github/workflows/vast.yaml

```
@@ -990,6 +990,19 @@ jobs:
           echo "::set-output name=tar_gz::${TAR_GZ}"
           echo "::set-output name=deb::${DEB}"
 
+      - name: Run Integration Tests
+        run: |
+          tar xf ./result/${{ steps.get_artifact_names.outputs.tar_gz }}
+          nix shell --command python vast/integration/integration.py --app opt/vast/bin/vast
+
+      - name: Upload Integration Test Logs on Failure
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: "vast-integration-test-nix-static-${{ matrix.build.compiler }}-${{ matrix.configure.tag }}"
```

There is no `matrix.build` and `matrix.configure` for the Nix jobs. For disambiguation in the matrix, I propose just using this:

```suggestion
          name: "vast-integration-test-nix-static-${{ matrix.nix.target }}"
```